README.md

```diff
@@ -57,3 +57,19 @@ const { firefox } = require("playwright-firefox");
 ## Best practises
 
 It's common to only install the [browser-specific NPM packages](https://playwright.dev/#version=v1.1.1&path=docs%2Finstallation.md&q=download-single-browser-binary), which will reduce installation time and slug size on Heroku in the end, that should fix also the error that the slug size is too large.
+
+Versions of `playwright-chromium@>1.15.0` take a new dependency on `playwright-core`, which does not play well with this buildpack, causing exceptions that look like
```

Comment:
Happy to land but I think we should replace his paragraph with https://github.com/playwright-community/heroku-playwright-buildpack/pull/20#issuecomment-1370253300.

Reply:
Thanks for the review. I tried a ton of variations of `playwright install chromium` but couldn't quite get it working on my project, but I can definitely remove the comment! Excited to get this merged

Reply:
Comment is updated